pyproject.toml

```diff
@@ -1,5 +1,123 @@
 [build-system]
-requires = ["setuptools >= 38.0.0",
-            "setuptools_scm >= 2.0.0",
-            "wheel"]
-build-backend = 'setuptools.build_meta'
\ No newline at end of file
+requires = ["setuptools >= 61.2",
+            "setuptools_scm[toml]>=7.1"]
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = "poppy"
+description = "Physical optics propagation (wavefront diffraction) for optical simulations, particularly of telescopes."
+authors = [{name = "Marshall Perrin", email = "mperrin@stsci.edu"}]
+license = {text = "BSD-3-Clause"}
+requires-python = ">=3.9"
+dependencies = [
+    "numpy>=1.20.0",
+    "scipy>=1.5.0",
+    "matplotlib>=3.2.0",
+    "astropy>=4.3.0",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+all = ["synphot"]
+test = [
+    "pytest",
+    "pytest-astropy",
+]
+docs = [
+    "nbsphinx",
+    "stsci_rtd_theme",
+    "sphinx-astropy",
+    "sphinx_rtd_theme",
+    "sphinx-automodapi",
+    "sphinx-issues",
+    "tomli; python_version <\"3.11\"",
+]
+
+[project.urls]
+Homepage = "https://github.com/spacetelescope/poppy"
+Documentation = "https://poppy-optics.readthedocs.io/"
+"Bug Tracker" = "https://github.com/spacetelescope/poppy/issues"
+
+[project.readme]
+text = '"POPPY (**P**\ hysical **O**\ ptics **P**\ ropagation in **Py**\ thon) is a Python package that simulates physical optical propagation including diffraction. It implements a flexible framework for modeling Fraunhofer and Fresnel diffraction and point spread function formation, particularly in the context of astronomical telescopes."'
```

Comment:
even easier, there's a shortcut in the `readme` keyword that lets it be just a filename string instead of a table; it would be `readme = "README.md"` under the `[project]` table

Reply:
changed, thanks!